### PR TITLE
[Chore #159071064] Adds and  Fixes tests for all GraphQL functionality.

### PR DESCRIPTION
**Pull request by parseendavid**
##### What Does This PR Do?
* Adds Tests for all available graphql functionality
* Fixed broken functionality and test snapshots.

#### How can this be manually tested?
* This can be tested by either running: 
` python server/manage.py test server/graphql_schemas/test`
or
`tox`

#### What are the pivotal tracker stories?
[#159071064](https://www.pivotaltracker.com/story/show/159071064)

### CODE REVIEW

File: server/graphql_schemas/tests/attend/test_mutations.py [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import logging
 
-from django.contrib.auth.models import User
-from api.models import Attend
+# from django.contrib.auth.models import User
```
Comment (tosmak16) [inline server/graphql_schemas/tests/attend/test_mutations.py]: @parseendavid Well done.... but please check commented code in line 3

Comment (parseendavid) [reply server/graphql_schemas/tests/attend/test_mutations.py]: I'll remove that.

File: server/graphql_schemas/event/schema.py [hunk 2]
```diff
@@ -22,34 +22,27 @@ class Input:
         title = graphene.String(required=True)
         description = graphene.String(required=True)
         venue = graphene.String(required=True)
-        date = graphene.String(required=True)
+        date = graphene.String(required=False)
```
[the remark this answers is not in the document]
Comment (parseendavid) [reply server/graphql_schemas/event/schema.py]: @tosmak16  As per the previous implementation(DRF api) and the model structure of an event, the `date` is not required field and can be edited later.

[the remark this answers is not in the document]
Comment (tosmak16) [reply server/graphql_schemas/event/schema.py]: @parseendavid Okay.

**Pull request merged**